config/development_active.yaml

```diff
@@ -102,3 +115,7 @@ namespaceDefaults:
 
 publicClient:
   hostPort: "localhost:7233"
+
+dynamicConfigClient:
+  filepath: "config/dynamicconfig/development_es.yaml"
+  pollInterval: "10s"
```

Comment:
nit: new line

Reply:
It is not there in any of configs. Why do you want it?

Reply:
It actually does present in few configs. I am removing it from everywhere.